client/jetpack-cloud/sections/partner-portal/lib/get-product-variant-short-title.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Format the string to return variant name
+ * @param product Product name
+ * @returns Product title
+ */
+export default function getProductVariantShortTitle( product: string ): string {
+	if ( product.includes( '10GB' ) ) {
+		return '10GB';
+	}
+
+	if ( product.includes( '100GB' ) ) {
+		return '100GB';
+	}
+
+	if ( product.includes( '1TB' ) ) {
+		return '1TB';
+	}
+
+	if ( product.includes( '3TB' ) ) {
+		return '3TB';
+	}
+
+	if ( product.includes( '5TB' ) ) {
+		return '5TB';
+	}
+
+	return product;
+}
```

Comment:
I think we can simplify this function:

```suggestion
export default function getProductVariantShortTitle(product: string): string {
  const match = product.match(/(\d+(?:GB|TB))/);
  return match ? match[1] : product;
}
```

WDYT?

Reply:
Make sense. nice suggestion 👍